File: 02-programacion-basica/02-variables-constantes-y-alcance/scala/ejemplos-guiados/VariablesYConstantes.scala
object VariablesYConstantes:
  def main(args: Array[String]): Unit =
    // val representa una referencia inmutable; es la opción preferida cuando no hay reasignación.
    val igv = 0.18
    val subtotal = 100.0
    val impuesto = subtotal * igv
    val total = subtotal + impuesto

    println(f"Subtotal: $subtotal%.2f | IGV: $impuesto%.2f | Total: $total%.2f")

File: 02-programacion-basica/02-variables-constantes-y-alcance/java/ejemplos-guiados/VariablesYConstantes.java
public class VariablesYConstantes {
    public static void main(String[] args) {
        // final indica que la referencia IGV no puede reasignarse.
        final double IGV = 0.18;
        double subtotal = 100.0;

        double impuesto = subtotal * IGV;
        double total = subtotal + impuesto;

        System.out.printf("Subtotal: %.2f | IGV: %.2f | Total: %.2f%n", subtotal, impuesto, total);
    }
}
